content/docs/hooks-reference.md

```diff
@@ -202,31 +204,59 @@ function Counter({initialCount}) {
   return (
     <>
       Count: {state.count}
-      <button onClick={() => dispatch({type: 'reset'})}>
-        Reset
-      </button>
       <button onClick={() => dispatch({type: 'increment'})}>+</button>
       <button onClick={() => dispatch({type: 'decrement'})}>-</button>
     </>
   );
 }
 ```
 
+#### Specifying the initial state
+
+There’s a few different ways to initialize `useReducer` state. You may choose either one depending on the use case. The simplest way to pass the initial state as a second argument:
+
+```js{3}
+  const [state, dispatch] = useReducer(
+    reducer,
+    {count: initialCount}
+  );
+```
+
+>Note
+>
+>React doesn’t use the `state = initialState` argument convention popularized by Redux. The initial value sometimes needs to depend on props and so is specified from the Hook call instead. If you feel strongly about this, you can write `state = initialState` both in the reducer and inside the `useReducer` destructuring assignment, but it's not encouraged.
```

Comment:
FWIW I found this very hard to parse due to my unfamiliarity. Maybe…

> To specify initial state, React doesn't use the default arguments pattern `function reducer(action, state = {count: 0})` popularized by Redux.

---

> you can write `state = initialState` both in the reducer and inside the `useReducer` destructuring assignment

can you not put it only in the reducer and let the initial state default to undefined? why?